Tests/Utils/PathLoader.swift

```diff
@@ -8,9 +8,36 @@
 import Foundation
 
 class PathLoader {
+
+    static let packageRoot: URL = {
+      // Will need to switch to “#filePath” after SE‐0274:
+      // https://github.com/apple/swift-evolution/blob/master/proposals/0274-magic-file.md
```

Comment:
**OffTopic**: Interesting. If I'm understanding it properly, it has implications for the issue that prevented me from being able to use `PythonKit` when I was experimenting with it. I couldn't add complex modules to the bundle because each submodule had an `__init__.py` file and Swift didn't like all those file with the same name being in the bundle.